Sources/LsbBitWriter.swift

```diff
@@ -54,7 +54,22 @@ public final class LsbBitWriter: BitWriter {
             mask <<= 1
         }
     }
+    
+    /**
+     Writes `number`, using and advancing by `bitsCount` BIT positions.
 
+     - Note: If `bitsCount` is smaller than the actual amount of `number`'s bits than the `number` will be truncated to
+     fit into `bitsCount` amount of bits.
+     - Note: Bits of `number` are processed using the same bit-numbering scheme as of the writer (i.e. "LSB 0").
+     - Note: this method is specifically useful when needing to write a UInt64 which can overflow and crash if converting to an Int when using the regular `write` method
+     */
+    public func write(unsignedNumber: UInt, bitsCount: Int) {
+        var mask: UInt = 1
+        for _ in 0..<bitsCount {
+            self.write(bit: unsignedNumber & mask > 0 ? 1 : 0)
+            mask <<= 1
+        }
+    }
```

Comment:
Something happened with empty lines here...

Reply:
New line is there, just below this and github doesn't recognize that the change is made.